github/resource_github_actions_runner_group_test.go

```diff
@@ -0,0 +1,274 @@
+package github
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccGithubActionsRunnerGroup(t *testing.T) {
+
+	randomID := acctest.RandStringFromCharSet(5, acctest.CharSetAlphaNum)
+
+	t.Run("creates runner groups without error", func(t *testing.T) {
+
+		// t.Skip("requires an enterprise cloud account")
```

Comment:
Since the tests don't run when this line is present, I wasn't sure if it should be there or not.